tests/test_auth_scopes.py

```
@@ -43,11 +69,12 @@ def test_env_overrides_toml_wo1162425_scopes(tmp_path: Path, monkeypatch) -> Non
     assert load_config().wo1162425_scopes is False
 
 
-def _cfg(*, wo1162425_scopes: bool) -> BlumkinConfig:
+def _cfg(*, wo1162425_scopes: bool, files_scopes: bool = False) -> BlumkinConfig:
     return BlumkinConfig(
         client_id="abc",
         config_dir=Path("/tmp/blumkin-test"),
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/blumkin-test"

(S108)